docker-compose.yml

```diff
@@ -55,6 +55,6 @@ services:
 
   # Third-party services
   mongodb:
-    image: mongo:3.4.4
+    image: mongo:3.6.0
```

Comment:
When services are updated like this, is it something that we will "get" automatically when we next run `make serve`, or do we need to explicitly `docker pull` this first?

https://github.com/foundry4/appeal-planning-decision/blob/master/Makefile#L16

e.g. 

```
serve:
	docker-compose pull && docker-compose up
.PHONY: serve
```

?

Reply:
If the images don't exist on your machine, `docker-compose up` does the `pull` beforehand - same as running `docker run some-image-what-i-dont-have`